src/panels/config/automation/dialog-new-automation.ts

```diff
@@ -30,59 +52,121 @@ class DialogNewAutomation extends LitElement implements HassDialog {
+    );
+  });
+
   protected render(): TemplateResult {
     if (!this._opened) {
       return html``;
     }
+
+    const processedBlueprints = this._processedBlueprints(this.blueprints);
+
     return html`
       <ha-dialog
         open
         hideActions
         @closed=${this.closeDialog}
         .heading=${createCloseHeading(
           this.hass,
-          this.hass.localize("ui.panel.config.automation.dialog_new.how")
+          this.hass.localize("ui.panel.config.automation.dialog_new.header")
         )}
       >
-        <mwc-list>
-          <mwc-list-item twoline class="blueprint" @click=${this._blueprint}>
-            ${this.hass.localize(
-              "ui.panel.config.automation.dialog_new.blueprint.use_blueprint"
-            )}
-            <span slot="secondary">
-              <ha-blueprint-picker
-                @value-changed=${this._blueprintPicked}
-                .hass=${this.hass}
-              ></ha-blueprint-picker>
-            </span>
-          </mwc-list-item>
-          <li divider role="separator"></li>
-          <mwc-list-item hasmeta twoline @click=${this._blank}>
+        <mwc-list
+          innerRole="listbox"
+          itemRoles="option"
+          innerAriaLabel=${this.hass.localize(
+            "ui.panel.config.automation.dialog_new.header"
+          )}
+          rootTabbable
+          dialogInitialFocus
+        >
+          <ha-list-item
+            hasmeta
+            twoline
+            graphic="icon"
+            @request-selected=${this._blank}
+          >
+            <ha-svg-icon slot="graphic" .path=${mdiPencilOutline}></ha-svg-icon>
             ${this.hass.localize(
-              "ui.panel.config.automation.dialog_new.start_empty"
+              "ui.panel.config.automation.dialog_new.create_empty"
             )}
             <span slot="secondary">
               ${this.hass.localize(
-                "ui.panel.config.automation.dialog_new.start_empty_description"
+                "ui.panel.config.automation.dialog_new.create_empty_description"
               )}
             </span>
             <ha-icon-next slot="meta"></ha-icon-next>
-          </mwc-list-item>
+          </ha-list-item>
+          <li divider role="separator"></li>
+          ${processedBlueprints.length > 0 ? html`` : null}
+          ${processedBlueprints.map(
+            (blueprint) => html`
+              <ha-list-item
+                hasmeta
+                twoline
+                graphic="icon"
+                @request-selected=${this._blueprint}
+                .path=${blueprint.path}
+              >
+                <ha-svg-icon
+                  slot="graphic"
+                  .path=${isHABlueprint(blueprint.path)
+                    ? mdiHomeAssistant
+                    : mdiAccount}
+                ></ha-svg-icon>
+                ${blueprint.name}
+                <span slot="secondary">${blueprint.path}</span>
```

Comment:
Should we show the path? Feels technical

Reply:
In my practice, a path's part - a filename - is used as the ONLY blueprint's name since in 99% cases I am using yaml to maintain automations. So, I suggest to display this filename.

Reply:
The file name is available in the blueprint page. This dialog is only for user that want to start a blueprint from UI.